docs/source/operator/credentials.rst

```diff
@@ -0,0 +1,56 @@
+.. _frankConsoleCredentials:
```

Comment:
Het gaat hier niet om credentials om op de console in te loggen, maar om credentials waarmee de Frank zichzelf naar andere systemen kan authenticeren

Reply:
Deze tekst ziet de lezer niet. Dit is een label in restructured text dat ik nodig heb om kruisverwijzingen te maken. Ik heb het label gekozen om consistent te zijn met andere labels in het hoofdstuk.

Reply:
Gerrit, je zei dat deze paragraaf misschien in het verkeerde hoofdstuk staat. Ik ga het verplaatsen naar het hoofdstuk voor administrators.

Reply:
Gedaan.